### PR TITLE
52 reformat dev according to pep8

### PR DESCRIPTION
**Pull request by ghost**
Edited code according to PEP8 for the whole project.

### CODE REVIEW

File: moninag/contact/views.py [hunk 1]
```diff
@@ -108,19 +109,19 @@ def post(self, request):
...
 
-        except:
+        except:  # pylint: disable=bare-except
             json_response['error'] = 'Invalid email format.'
             return JsonResponse(json_response, status=400)
 
-    def verify(request, activation_key):
-
+    def verify(request, activation_key):  # pylint: disable=no-self-argument
+        """Making contact active (active=False -> active=True)"""
```
Comment (nkoblai) [inline moninag/contact/views.py]: Missing blank space after docstring.

**Review by nkoblai — APPROVED** [review]
@HenrikhB it works.

File: moninag/home/views.py [hunk 2]
```diff
@@ -8,4 +10,7 @@ class IndexView(View):
     """
 
     def get(self, request):
+        """
+        Method which renders main index page
+        """
```
Comment (hutsulyk) [inline moninag/home/views.py]: Move to one line :slightly_smiling_face:

File: moninag/nagplugin/models.py [hunk 3]
```diff
@@ -27,8 +29,8 @@ def get_by_id(id):
         """
 
         try:
-            plugin = NagPlugin.objects.get(id=id)
-        except Exception as error:
+            plugin = NagPlugin.objects.get(id=plugin_id)
+        except Exception as error:  # pylint: disable=broad-except, unused-variable
```
Comment (hutsulyk) [inline moninag/nagplugin/models.py]:
```
except NagPlugin.DoesNotExist:
```
:upside_down_face:

File: moninag/registration/views.py [hunk 4]
```diff
@@ -79,11 +90,14 @@ def register_user(request):
 
                 user.save()
                 json['success'] = True
-                json['message'] = "Thank you for your time. The confirmation code has been sent to your email. In order to confirm the registration, simply click on the link given in it."
+                json[
+                    'message'] = "Thank you for your time. The confirmation code has" \
```
Comment (hutsulyk) [inline moninag/registration/views.py]: Why `message` on new line? :space_invader:

Comment (ghost) [reply moninag/registration/views.py]: fixed 👹

File: moninag/contact/views.py [hunk 5]
```diff
@@ -108,19 +109,20 @@ def post(self, request):
                 json_response['response'] = contact.to_dict()
                 return JsonResponse(json_response, status=201)
 
-        except:
+        except:  # pylint: disable=bare-except
```
Comment (hutsulyk) [inline moninag/contact/views.py]: `ValidationError` I guess :slightly_smiling_face:

File: moninag/contact/views.py [hunk 6]
```diff
@@ -159,11 +161,11 @@ def put(self, request, contact_id):
             json_response['response'] = contact.to_dict()
             return JsonResponse(json_response, status=200)
 
-        except:
+        except:  # pylint: disable=bare-except
```
Comment (hutsulyk) [inline moninag/contact/views.py]: `ValidationError` :upside_down_face:

File: moninag/daemons/check_launcher.py [hunk 7]
```diff
@@ -47,7 +47,7 @@ def inner(*args, **kwargs):
             while mtries:
                 try:
                     return func(*args, **kwargs)
-                except Exception as error:
+                except Exception as error:  # pylint: disable=broad-except, unused-variable
```
Comment (hutsulyk) [inline moninag/daemons/check_launcher.py]: `as error` could be removed

File: moninag/registration/views.py [hunk 8]
```diff
@@ -79,11 +90,13 @@ def register_user(request):
...
                 json['success'] = True
-                json['message'] = "Thank you for your time. The confirmation code has been sent to your email. In order to confirm the registration, simply click on the link given in it."
+                json['message'] = "Thank you for your time. The confirmation code has" \
+                                  " been sent to your email. In order to confirm the " \
+                                  "registration, simply click on the link given in it."
 
                 send_activation_email(
                     DEFAULT_HOST, DEFAULT_FROM_EMAIL, user.email, activation_key)
-        except:
+        except:  # pylint: disable=bare-except
```
Comment (hutsulyk) [inline moninag/registration/views.py]: ValidationError :space_invader:

File: moninag/registration/views.py [hunk 9]
```diff
@@ -111,22 +126,26 @@ def request_password_reset(request):
...
                 json['success'] = True
-                json['message'] = "An email has been sent to " + user.email +". Please check its inbox to continue reseting password."
+                json['message'] = "An email has been sent to " + user.email + \
+                                  ". Please check its inbox to continue reseting password."
 
             except CustomUser.DoesNotExist:
                 json['error'] = "No user is associated with this email address"
 
-        except:
+        except:  # pylint: disable=bare-except
```
Comment (hutsulyk) [inline moninag/registration/views.py]: ValidationError :horse:

File: moninag/registration/views.py [hunk 10]
```diff
@@ -147,7 +167,9 @@ def confirm_password_reset(request, uidb64=None, token=None):
             new_password = data.get('password')
             user.set_password(new_password)
             user.save()
-            json['message'] = "Password has been reset. In 5 seconds you will be redirected to the login page."
+            json[
+                'message'] = "Password has been reset. In 5 seconds you will be redirected" \
```
Comment (hutsulyk) [inline moninag/registration/views.py]: Move to one line :robot:

**Review by hutsulyk — APPROVED** [review]
Great job! :+1:

**Pull request merged**